source/openapi-admin-v3.yaml

```diff
@@ -2596,9 +2599,51 @@ paths:
                 type: array
                 items:
                   "$ref": "#/components/schemas/Deployment"
+  "/groups/{groupId}/apps/{appId}/deployments/{deploymentId}":
+    parameters:
+      - "$ref": "#/components/parameters/GroupId"
+      - "$ref": "#/components/parameters/AppId"
+      - "$ref": "#/components/parameters/DeploymentId"
+    get:
+      tags:
+        - deploy
+      operationId: adminGetDeployment
+      summary: Get a deployment
+      description: Get information about a recent deployment of the application.
+      responses:
+        "200":
+          description: Ok
+          content:
+            application/json:
+              schema:
+                "$ref": "#/components/schemas/Deployment"
+  "/groups/{groupId}/apps/{appId}/deployments/{deploymentId}/redeploy":
     parameters:
       - "$ref": "#/components/parameters/GroupId"
       - "$ref": "#/components/parameters/AppId"
+      - "$ref": "#/components/parameters/DeploymentId"
+    get:
+      tags:
+        - deploy
+      operationId: adminRedeployDeployment
+      summary: Redeploy a deployment
+      description: Redeploy a previously deployed deployment.
+      responses:
+        "204":
+          description: Successfully redeployed
+        "404":
+          description: Encountered an error while updating the trigger.
+          content:
+            application/json:
+              schema:
+                "$ref": "#/components/schemas/Error"
+                properties:
```

Comment:
This error information doesn't seem to be rendering on the page.

Reply:
Oops - missing an `allOf` here, good catch!